README.md

```diff
@@ -202,6 +202,11 @@ cloudinary.image("sample", {format: "png", width: 100, height: 100, crop: "fill"
 // <img src='http://res.cloudinary.com/demo/image/upload/c_fill,h_100,w_100/sample.png' height='100' width='100'/>
 ```
 
+### Typescript
+
+We added support for TS, The SDK can now be consumed in a TS environment.
+You can view the decleration file [here](https://github.com/cloudinary/cloudinary_npm/types/index.d.ts).  
+
```

Comment:
Would rephrase to:
"🎉New 🎉TypeScript support was just added. Check out the [declaration file] (URL) "